Basic-Car-Maintenance/Shared/Models/Vehicle.swift

```diff
@@ -14,4 +14,8 @@ struct Vehicle: Codable, Identifiable {
     let name: String
     let make: String
     let model: String
+    let year: String
+    let color: String
+    let VIN: String
+    let licensePlateNumber: String
```

Comment:
```suggestion
    let year: String?
    let color: String?
    let VIN: String?
    let licensePlateNumber: String?
```

These should be optional